.Dockerignore

```diff
@@ -0,0 +1 @@
+.tox
```

Comment:
I'll add this to the template soon(tm): but also add a line `.runner` here as well?